addons/html_editor/static/src/main/link/link_popover.js

```diff
@@ -210,7 +211,10 @@ export class LinkPopover extends Component {
             const externalMetadata = await this.props.getExternalMetaData(this.state.url);
 
             this.state.urlTitle = externalMetadata?.og_title || this.state.url;
-            this.state.urlDescription = externalMetadata?.og_description || "";
+            this.state.urlDescription = externalMetadata?.og_description
+                ? html_parser.parseFromString(externalMetadata.og_description, "text/html").body
```

Comment:
I have a feeling that this conversion should be done by the server instead in `link_preview_metadata`.

Reply:
Hello @dmo-odoo, I've done some changes in `link_preview_metadata` to handle this. Can you please have a look?